src/effects/VST3/VST3Effect.cpp

```diff
@@ -562,8 +397,11 @@ void VST3Effect::ExportPresets(const EffectSettings& settings) const
       );
       return;
    }
-   
-   if (!mWrapper->SavePreset(fileStream))
+
+   auto wrapper = std::make_unique<VST3Wrapper>(*mModule, mEffectClassInfo.ID());
+   wrapper->FetchSettings(settings);
```

Comment:
Ninth commit re-introduces use of the `settings` argument, good

Reply:
And also it creates a helper effect instance